src/assignment/views/AssignmentOverview.tsx

```diff
@@ -500,16 +500,7 @@ const AssignmentOverview: FunctionComponent<AssignmentOverviewProps> = ({
 			}
 
 			case 'labels':
-				return labels.length > 0 ? (
-					<>
-						{renderLabels(labels)}
-						{renderDataCell(
-							labels.map((label) => label.assignment_label.label).join(', '),
-							`${t('assignment/views/assignment-overview___labels')}`,
-							'm-assignment-overview__table__data-cell--labels'
-						)}
```

Comment:
This is used for mobile layout, should remain intact.
see `src/assignment/views/AssignmentOverview.scss`

Reply:
fixed: 
![image](https://user-images.githubusercontent.com/1710840/188847955-f4a60ca1-8479-4d0e-ae00-881dc841ac49.png)